schema/revert/data/correct_gwp_values.sql

```diff
@@ -0,0 +1,82 @@
+-- Revert ggircs-portal:data/correct_gwp_values from pg
+
+begin;
+
+alter table ggircs_portal.form_result disable trigger _100_timestamps, disable trigger _immutable_form_result;
+
+  -- CH4
+update ggircs_portal.form_result
+  set form_result = replace(form_result::text, '"gwp": 28', '"gwp": 25')::jsonb
+  where application_id in (
+    select application_id
+    from ggircs_portal.application_revision
+    where created_at < '2023-05-30' and created_at > '2023-04-01'
```

Comment:
we probably want that selector on application year on the revert file (or just not implement revert?)